test/test_linalg.py

```diff
@@ -3204,6 +3204,14 @@ def test_inv_ex(input, out=None):
                         batches, n
                     )
 
+    @skipCUDAIfNoMagmaAndNoCusolver
+    @skipCPUIfNoLapack
+    @dtypes(torch.float32, torch.float64, torch.complex64, torch.complex128)
+    def test_inv_ex_info_device(self, device, dtype):
```

Comment:
Add a test to catch the small error that we found above, just in case, although it it's too difficult to trigger consistently (pass the matrix with all zeroes?) it's fine.

Reply:
`info` exposed here is the `info_lu` in the above code. We don't expose "info_getri" because it's not useful.